frontend/fingertips-frontend/app/chart/page.tsx

```diff
@@ -0,0 +1,27 @@
+import { LineChart } from '@/components/pages/chart';
+import {connection} from "next/server";
+
+async function fetchLineChartData() {
+  // We don't want to render this page statically
+  await connection();
+  
+  const response = await fetch('http://localhost:5144/weatherforecast', {
+    // Cache the data for 60s
+    next: { revalidate: 60 },
+  });
+  
+  if (!response.ok) {
+    throw new Error('Failed to fetch forecast data');
+  }
+  return await response.json();
+}
+
+export default async function getLineChartData() {
```

Comment:
I don't think this function should be called `getLineChartData`, as it's the main function for this page. Perhaps `ChartPage`?

Reply:
Also, can we not use a default export here please? As [documented in our new coding standards](https://confluence.collab.test-and-trace.nhs.uk/display/FTN/Coding+Standards#CodingStandards-UseNamedExportsRatherthanDefaultExports) (I appreciate I haven't publicised them yet!).

Reply:
It can be called ChartPage, I don't have access to confluence pages yet. Could you send me a pdf of the coding standards please?